src/main/java/org/asf/centuria/interactions/NetworkedObjects.java

```diff
@@ -40,67 +40,109 @@ public static void init() {
 					.getResourceAsStream("networkedobjects.json");
 			JsonObject helper = JsonParser.parseString(new String(strm.readAllBytes(), "UTF-8")).getAsJsonObject();
 			strm.close();
+			loadObjects(helper);
 
-			// Load all entries
+			// Load transformers
+			strm = InventoryItemDownloadPacket.class.getClassLoader()
+					.getResourceAsStream("objecttransformers/index.json");
+			JsonArray transformers = JsonParser.parseString(new String(strm.readAllBytes(), "UTF-8")).getAsJsonArray();
+			strm.close();
+			for (JsonElement el : transformers) {
+				strm = InventoryItemDownloadPacket.class.getClassLoader()
+						.getResourceAsStream("objecttransformers/" + el.getAsString());
+				helper = JsonParser.parseString(new String(strm.readAllBytes(), "UTF-8")).getAsJsonObject();
+				strm.close();
+				loadObjects(helper);
+			}
+		} catch (IOException e) {
+		}
```

Comment:
Please log any exceptions you run into, or put a comment saying why you are suppressing a particular exception. Suppression like this can create problems for future debugging work.

Reply:
oops forgot that

Reply:
ill do a runtime exception as it will break the server if it doesnt get thrown